src/workers/issues-status-message-builder.ts

```diff
@@ -0,0 +1,61 @@
+import { Issue } from '../services/jira-service';
+
+export interface IssuesStatusMessageBuilder {
+  buildMessage(
+    issuesPerProjectKeyAndStatus: Map<string, Map<string, Issue[]>>
+  ): string;
+}
+
+export class ConcreteIssuesStatusMessageBuilder
+  implements IssuesStatusMessageBuilder {
+  private readonly jiraHost: string;
+
+  constructor(jiraHost: string) {
+    this.jiraHost = jiraHost;
+  }
+
+  buildMessage(
+    issuesPerProjectKeyAndStatus: Map<string, Map<string, Issue[]>>
```

Comment:
There is a dependency of Issue being a Jira issue here, maybe introduce an interface of Issue